openapi/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "y-sweet-openapi-doc",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "mkdir -p build && redocly build-docs server.yml --output build/index.html && redocly build-docs document.yml --output build/document.html"
```

Comment:
Might be a good idea to clean the directory before running build, too? `rm -r build && mkdir build && ...`